identity-apps-core/apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -395,6 +418,12 @@
     %>
 <% } %>
 
+<% if (isAdminBannerAllowedInSP && isAdminAdvisoryBannerEnabledInTenant) { %>
```

Comment:
Do we need this `&&` condition here ? Can't we use only 
```
if (isAdminAdvisoryBannerEnabledInTenant) {
```

Reply:
Yes, since `isAdminAdvisoryBannerEnabledInTenant` will always be false if `isAdminBannerAllowedInSP` is set to false.

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/4114/commits/c3cdf6db739c3e0e0ebf0139fdb1cd714f472897